src/css/customized/vue-multiselect.css

```diff
@@ -168,9 +168,8 @@ fieldset[disabled] .multiselect {
   background: #8C215C;
   margin-bottom: 5px;
   white-space: normal;
-  overflow: hidden;
   max-width: 100%;
-  text-overflow: ellipsis;
+  overflow-wrap: anywhere;
```

Comment:
I've removed 
```
overflow: hidden;
text-overflow: ellipsis;
```
Hope that's okay.

Reply:
I'm curious how the wrapping is handled in the design system components, but I think it's OK to have this change to unblock the release testing.

Reply:
🤔 I'm not sure I understand, let's discuss!
We are not using the design system components with our selectors.
Do you mean that we should update the selectors in `viamo-ui` in the same fashion as in this PR?